source/openpulse/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.black]
+line-length = 100
+target-version = ["py36"]
```

Comment:
Since we're using `dataclasses`, we must be using Python 3.7 at least.

Reply:
I updated the one under openpulse but left the one under oepnqasm intact. That one is also referencing py36.